### PR TITLE
Implemented exponential backoff timeouts

### PR DESCRIPTION
**Pull request by amwolff**
Jobs now have exponential backoff timeouts.

### CODE REVIEW

File: cmd/example_usage.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package main
 import (
 	"html/template"
 
+	"github.com/cenkalti/backoff"
+
 	"github.com/datainq/jobhub"
```
Comment (orian) [inline cmd/example_usage.go]: Please group external imports together

File: jobhub.go [hunk 2]
```diff
@@ -7,6 +7,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/cenkalti/backoff"
+
```
Comment (orian) [inline jobhub.go]: ditto

**Review by orian — CHANGES_REQUESTED** [review]
Just a tiny changes required.

File: cmd/example_usage.go (hunk 1 above)
Comment (amwolff) [reply cmd/example_usage.go]: Done.

File: jobhub.go (hunk 2 above)
Comment (amwolff) [reply jobhub.go]: Done.

**Review by orian — APPROVED** [review]
well done

**Pull request merged**